### DISCUSSION TITLE
Allow changing time format irrespective of browser language

### DISCUSSION THREAD

**Discussion by ecksun** [home-assistant/frontend#8392] [Localization]
I would like to be able to set the time and date format independent of what language I have configured in my browser.

Currently my browser is set to english, yet I would like to use 24-hour time format and ISO dates (YYYY-MM-DD).

Having my browser set to english is both something I prefer but also helps improve privacy as any other language would make my browser fingerprint more unique.

**Comment by paulius2k** [comment]
Currently (core version `2022.3.2`) it is possible to select the `Time format` irrespective of the selected language.
BUT not the `Date format`. I have registered https://github.com/home-assistant/frontend/discussions/11973
